packages/util-package-renderer/lib/js/sass-helper.js

```diff
@@ -34,12 +31,11 @@ const compileSASS = async (packageRoot, demoCodeFolder, minify) => {
 
 	// Render the SASS to CSS
 	try {
-		result = await render({
-			data: packageSASS,
-			outputStyle: (minify) ? 'compressed' : 'expanded',
+		result = sass.compileString(packageSASS, {
+			style: (minify) ? 'compressed' : 'expanded',
 			indentType: 'tab',
 			indentWidth: 1,
-			includePaths: [
+			loadPaths: [
```

Comment:
this is one of my annoyances with Dart Sass. The need to have similar things named slightly differently.

Reply:
Yep, there are a few bits like that, but the docs are generally pretty decent